vr/components/Widgets/ImageWidget.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { View, Text, StyleSheet, staticAssetURL, Image, VrButton } from "react-360";
+import VideoBackground from "../Background/VideoBackground";
+import LabelButton from "./../Label/LabelButton";
+import BackgroundManager from "./../Background/BackgroundManager";
+
+const styles = StyleSheet.create({
+    root: {
+        position: 'absolute',
+        layoutOrigin: [-0.5, -0.5],
+    },
+    text: {
+        color: 'black',
+    }
+});
+
+export default class ImageWidget extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            fullScreen: false
+        }
+    }
+
+    sizes = {
+        width: this.props.widget.width,
+        height: this.props.widget.height
+    }
+
+    _handleImageClick = () => {
+        const { fullScreen } = this.state;
+        const { widget } = this.props;
+        this.setState({fullScreen: !fullScreen});
+
+        if (!fullScreen) {
+            this.sizes = {
+                height: 600,
+                width: 600 * (widget.width / widget.width), //Proportional increase
+            }
+        } else {
+            this.sizes = {
+                width: this.props.widget.width,
+                height: this.props.widget.height
+            }
+        }
+    }
+
+    render() {
+        const { widget } = this.props;
+
+        return <View style={[styles.root, { transform: [{ translate: [widget.x, this.state.fullScreen ? 300 : widget.y], }] }]}>
+            {widget.image ? 
+                <View>
+                    <Image 
+                        source={{uri: `http://localhost:3001/${widget.image.filename}`}}
```

Comment:
put it here
![image](https://user-images.githubusercontent.com/9947582/59342921-6c288280-8d13-11e9-9fc0-6647607c7412.png)